File: examples/db/dropTable/go/dropTable.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/db"
)

// Drop a table in the DB
func DropTable() {
	dbService := db.NewDbService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := dbService.DropTable(&db.DropTableRequest{
		Table: "users",
	})
	fmt.Println(rsp, err)
}

File: examples/db/dropTable/node/dropTable.js
const { DbService } = require("micro-js-client/db");

// Drop a table in the DB
async function dropTable() {
  let dbService = new DbService(process.env.MICRO_API_TOKEN);
  let rsp = await dbService.dropTable({
    table: "users",
  });
  console.log(rsp);
}

dropTable();
